scipy/sparse/_csr.py

```diff
@@ -276,6 +274,14 @@ def _get_arrayXslice(self, row, col):
             return self._get_arrayXarray(row, col)
         return self._major_index_fancy(row)._get_submatrix(minor=col)
 
+    def _set_int(self, idx, x):
+        major, minor = self._swap((0, idx))
```

Comment:
Since we only support 1d for CSR format, this can be simplified to `self._set_many(0, idx, x)`